jupyterlite_sphinx/jupyterlite_sphinx.py

```diff
@@ -522,7 +522,7 @@ def run(self):
             ] = notebook_unique_name
             # Copy the Notebook for NotebookLite to find
             os.makedirs(notebooks_dir, exist_ok=True)
-            with open(notebooks_dir / Path(notebook_unique_name), "w") as f:
+            with open(notebooks_dir / Path(notebook_unique_name), "w", encoding="utf-8") as f:
                 # nbf.write incorrectly formats multiline arrays in output.
                 json.dump(nb, f, indent=4, ensure_ascii=False)
```

Comment:
An alternative, given you're using Pathlib:
```suggestion
            # nbf.write incorrectly formats multiline arrays in output.
            nb_json = json.dump(nb, f, indent=4, ensure_ascii=False)
            (notebooks_dir / notebook_unique_name).write_text(nb_json, encoding="utf-8")
```

Reply:
We have `nbformat` as a dependency now. Can we make it write the notebook here instead of having to rely on JSON directly?

Reply:
Yep sorry - I just made the suggestion via GH's editor, missed checking it properly.

A

Reply:
Ah, thanks for the context, @steppi. I thought the problem might have been resolved by now, but that's something to look for in a different place.